docker/Dockerfile.binary

```diff
@@ -0,0 +1,26 @@
+FROM docker.io/clux/muslrust:stable as cargo-build
+WORKDIR /usr/src/gpdata
+
+# Copy and Build Code
+COPY . .
+RUN env CARGO_PROFILE_RELEASE_DEBUG=1 cargo build --target x86_64-unknown-linux-musl --release
```

Comment:
Curious as to why you want to include debug information in the release build.

Reply:
You are right, Probably, it is not needed. Though I thought it is a good practice and I just "copied it" from gp-v2-services